c-bindings-gen/src/main.rs

```diff
@@ -1342,6 +1351,14 @@ fn convert_file<'a, 'b>(libast: &'a FullLibraryAST, crate_types: &mut CrateTypes
 		let mut out = std::fs::OpenOptions::new().write(true).create(true).truncate(true)
 			.open(new_file_path).expect("Unable to open new src file");
 
+		writeln!(out, "// This file is Copyright its original authors, visible in version control").unwrap();
+		writeln!(out, "// history and in the source files from which this was generated.").unwrap();
+		writeln!(out, "//").unwrap();
+		writeln!(out, "// This file is licensed under the license available in the LICENSE or LICENSE.md").unwrap();
+		writeln!(out, "// file in the root of this repositry or, if no such file exists, the same license").unwrap();
+		writeln!(out, "// as that which applies to the original source files.").unwrap();
```

Comment:
nit: repository

Were these sentences pre-ordained by some lawyer somewhere? Just wondering because I'm a bit confused whether "the original souce files" refers to Rust-Lightning or `lightning-c-bindings`, but if it's preapproved legalese that's fine..

Reply:
Nope, you can tell I wrote it because there's typos in it :).